src/jot.gleam

```diff
@@ -437,6 +569,12 @@ fn parse_codeblock_content(
   }
 }
 
+/// get all content up until the end of the line erasing up to `indentation`
+/// spaces.
+///
+/// Note:
+/// If indentation is not fully satisfied by preceding spaces duplicated spaces
+/// will be removed.
```

Comment:
What does this mean? Could you give an example and explain why it matters 🙏 

Capital letters at the start of sentences please.